netbox/dcim/api/serializers_/cables.py

```diff
@@ -42,16 +42,16 @@ class TracedCableSerializer(serializers.ModelSerializer):
     Used only while tracing a cable path.
     """
     url = serializers.HyperlinkedIdentityField(view_name='dcim-api:cable-detail')
+    display_url = serializers.HyperlinkedIdentityField(view_name='dcim:cable')
```

Comment:
This should be updated to inherit from BaseModelSerializer, and add a `display` field. (Or, alternatively, leaving it in its original state, without adding a `display_url` field.)